proto/pulumi/provider.proto

```diff
@@ -48,7 +48,15 @@ service ResourceProvider {
     rpc CheckConfig(CheckRequest) returns (CheckResponse) {}
     // DiffConfig checks the impact a hypothetical change to this provider's configuration will have on the provider.
     rpc DiffConfig(DiffRequest) returns (DiffResponse) {}
+
     // Configure configures the resource provider with "globals" that control its behavior.
+    //
+    // Warning:
+    //
+    //     ConfigureRequest.args may include secrets. Because ConfigureRequest is sent before
+    //     ConfigureResponse can specify acceptSecrets: false, providers *must* handle secrets from
+    //     ConfigureRequest.args.
+    //
```

Comment:
If we want to have some fun, we could try:

```
:::{warning}
ConfigureRequest.args ...
:::
```

Then you'll get a pretty admonition at https://pulumi-developer-docs--17348.org.readthedocs.build/17348/docs/_generated/proto/provider.html#configure, and I don't think the in-code readability cost is too high? But happy to be told to shut up :joy:

Reply:
Seems sensible to me. Reads as well as Warning: indent

Reply:
Shame there isn't a cross language way to do this really cleanly :(

Reply:
I'm happy to try it.